.github/workflows/changes-in-high-risk-code.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/checkout@v4
       - name: Get changed files
         id: changes
-        uses: getsentry/paths-filter@4512585405083f25c027a35db413c2b3b9006d50 # v2.11.1
+        uses: dorny/paths-filter@0bc4621a3135347011ad047f9ecf449bf72ce2bd # v3.0.0
```

Comment:
Why are we moving await from Sentry's `paths-filter`, if we need functions that it does not offer should we propose an update?